robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot

```diff
@@ -7,7 +7,7 @@ Library        cumulusci.robotframework.PageObjects
 Suite Setup     Run Keywords
 ...             Open Test Browser
 ...             Setup Test Data
-Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+#Suite Teardown  Capture Screenshot and Delete Records and Close Browser
```

Comment:
Add Teardown method

Reply:
Looks like the same file as above, added suite teardown back.